### ISSUE TITLE
Flaming Cliffs: aircraft do not get live data updates

### ISSUE THREAD

**Issue by Durrman78** [DCS-Skunkworks/dcs-bios#920]
### Version

- [X] I am using the latest version

### Aircraft

FC3

### Control

Altitude

### Description

The status in Bort is flashing yellow, not green.  I do not see any live data for any value for Flaming Cliffs aircraft.  I checked A-10A, F86, and F5-E.

F18 behaved correctly.

### Steps to reproduce

Load into A-10A
Open Bort.
See no live data.

### Expected behavior

Expecting to see live data on the available FC3 outputs.

### Screenshots

![image](https://github.com/user-attachments/assets/4fde2433-9c5f-472b-90e3-7e6bfa0d4819)

**Comment by charliefoxtwo** [comment]
I investigated this a little bit, and there are two issues here:

1) the new FC2024 aircraft need to be added to bios. This is an easy fix.
2) at some point DCS stopped providing us with a certain scripting component only for FC3 aircraft (it's still present for FC2024 aircraft) (`GetDevice(0)` returns nil, for those curious). I've filed a bug report with ED, but I think we can work around this in the meantime as this component isn't actually required for FC exports - just for full-fidelity modules.
